src/raster/r.survey/r.survey.html

```diff
@@ -0,0 +1,290 @@
+
+<p>
+<p>
+	
+3D Distance is the three dimensional linear distance between a viewpoint and a target pixel (see the illutratio bellow). The min3dDistance map portrays  the value of the minimum three-dimensional distance between each pixel and the closest viewpoint, in meters. 
+<p>
+	
+<img src="r.survey_figures/fig_2.png" alt="e3Ddistance">
+
+<p>
+Given a viewpoint, r.survey calculate the unit vector describing direction and sense of the line of sight between that viewpoint and each visible pixel (see illustration bellow). We define View Angle as the angle between the line of sight unit vector and the versor normal to the terrain surface in each pixel. The maxViewAngle map shows the value of the maximum View Angle between each pixel and the viewpoints, in degrees. It is a measure of the most frontal view each single cell is visible from.  View angle output is always larger than 90\B0 and smaller than 180\B0.
+<p>
+	
+<img src="r.survey_figures/View_Angle.png" alt="view angle">
+
+
+<p>
+Solid Angle is one of the best and most objective indicators for quantifying the visibility. The idea is that any observed object will be progressively less apreciable the more far away and the more tilted it is with regard to the viewpoint. As a consequence Solid Angle depends on the size of the observed object as well as on the distance and the orientation from where this object is observed.
+Solid Angle of a surface is, by definition, equal to the projected spherical surface in the evolving sphere divided by the square of the radius of the sphere. As an aproximation, we use the ellipse surface area in place of the projected spherical surface in the evolving sphere. The maxSolidAngle map shows the value of the maximum Solid Angle among those measured from the different viewpoints. In this map, the closest pixels to the observation points have to be interpreted with special attention, considering that the error, respect to the real Solid Angle, can reach until the 10% in the immediate neighbor pixels
+<p>
+Other three maps (pointOfViewWithMin3dDistance, pointOfViewWithMmaxAngle and pointOfViewWithMmaxSolidAngle) are used to register, in each cell, the identifier of the viewpoints from where an observer can get, respectively, (i) the minimum values of 3D Distance, (ii) the maximum values of View Angle and (iii) the maximum value of the Solid Angle. Another relevant output is the numberOfViews map which portraits the number of viewpoints from where each pixel is visible.
+
+<h2>INSTALLATION</h2>
+<h3>LINUX:</h3>
+
+1) Download the compressed folder of r.survey from <a href="https://doi.org/10.5281/zenodo.3993140">https://doi.org/10.5281/zenodo.3993140</a> and decompress it. 
+<p>
+2) Copy the file <em>r.survey.py</em> into the folder <em>/usr/lib/grass78/scripts/</em>
+<p> 
+3) Copy the file <em>r.survey.html</em> and the folder <em>r.survey_figures</em> into the directory <em>/usr/lib/grass78/docs/html/</em>
+<p> 
```

Comment:
I'd delete all this - it shall be installed correctly with `g.extension`.

Reply:
Yes.. I'll do it